tox.ini

```diff
@@ -10,44 +10,34 @@ envlist = lint, unit
 src_path = {toxinidir}/src/
 tst_path = {toxinidir}/tests/
 all_path = {[vars]src_path} {[vars]tst_path}
-lxd_name = slurmrestd-func-test
-ftest_path = {toxinidir}/tests/functional
 
 [testenv]
 setenv =
-  PYTHONPATH = {toxinidir}:{toxinidir}/lib:{[vars]src_path}
-  PYTHONBREAKPOINT=ipdb.set_trace
-  PY_COLORS=1
+    PYTHONPATH = {toxinidir}:{toxinidir}/lib:{[vars]src_path}
+    PYTHONBREAKPOINT=ipdb.set_trace
+    PY_COLORS=1
 passenv =
-  PYTHONPATH
-  CHARM_BUILD_DIR
-  MODEL_SETTINGS
+    PYTHONPATH
+    CHARM_BUILD_DIR
+    MODEL_SETTINGS
 
 [testenv:fmt]
 description = Apply coding style standards to code
 deps =
-    black==22.12.0
-    ruff==0.0.206
+    black
+    ruff
 commands =
     ruff --fix {[vars]all_path}
     black {[vars]all_path}
```

Comment:
Wanted to ask this to you. If ruff fails, have you noticed that black does not apply any formatting changes to the source code? I noticed that so I changed the execution order of ruff and black. I have the order like the following now:

```ini
commands =
    black {[vars]all_path}
    ruff --fix {[vars]all_path}
```

Let me know if you see the same thing that I am seeing!

Reply:
Yes, it's happened to me too. Will change the ordering too

Reply:
Just pushed an update applying this reorder